### ISSUE TITLE
Size!

### ISSUE THREAD

**Issue by crysan** [L4Digital/FastScroll#2]
Hi
Pls add size bubble setting.
Ty!

**Comment by randr0id** [comment]
We might add attribute support for this in the future, however, you can override any of the dimension values from FastScroll library's [`dimens.xml`](https://github.com/L4Digital/FastScroll/blob/master/fastscroll/src/main/res/values/dimens.xml). 

For the bubble size, add `<dimen name="fastscroll_bubble_radius">XXdp</dimen>` to your project's `dimens.xml`.

**Comment by crysan** [comment]
The future is now.
Add this function to the library.

**Comment by randr0id** [comment]
After further review, the best method for adjusting the bubble dimensions is by overriding these values:
~~~xml
<dimen name="fastscroll_bubble_radius">44dp</dimen>
<dimen name="fastscroll_bubble_size">88dp</dimen>
<dimen name="fastscroll_bubble_textsize">48sp</dimen>
~~~

**Comment by NainalChauhan** [comment]
I want to have two different bubble textsize in two layouts. By overriding fastscroll_bubble_textsize it will change the textsize for whole project. Is there any way to change it programmatically?

**Comment by randr0id** [comment]
There isn't currently, but I'm working to add this and the other bubble dimensions to the upcoming [2.0 release](https://github.com/L4Digital/FastScroll/releases/tag/2.0.0-beta1).

**Comment by randr0id** [comment]
I added attributes and methods to change the bubble text size. I also added an attribute to change the bubble size to either "normal" or "small". These changes are released in [2.0.0-beta2](https://github.com/L4Digital/FastScroll/releases/tag/2.0.0-beta2).
